From 6c295719c5ea88fbf2987d5109ddc934248fb9c9 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (VMware)" <rostedt@goodmis.org>
Date: Mon, 17 Jun 2019 16:05:29 -0400
Subject: [PATCH 1/2] kernel-shark: Have "make clean" run cmake-clean.sh

commit 9aef575e9acdbae851227655d73dff7ba0aed45f upstream

A make clean should clean up the cmake files as well.

Reviewed-by: Yordan Karadzhov <ykaradzhov@vmware.com>
Signed-off-by: Steven Rostedt (VMware) <rostedt@goodmis.org>
Signed-off-by: Sudip Mukherjee <sudipm.mukherjee@gmail.com>
---
 Makefile            | 1 +
 kernel-shark/README | 1 +
 2 files changed, 2 insertions(+)

diff --git a/Makefile b/Makefile
index 2bcc383..c2fe8f8 100644
--- a/Makefile
+++ b/Makefile
@@ -368,6 +368,7 @@ clean:
 	$(MAKE) -C $(src)/python clean
 	$(MAKE) -C $(src)/tracecmd clean
 	if [ -f $(kshark-dir)/build/Makefile ]; then $(MAKE) -C $(kshark-dir)/build clean; fi
+	cd $(kshark-dir)/build; ./cmake_clean.sh
 
 
 ##### PYTHON STUFF #####
diff --git a/kernel-shark/README b/kernel-shark/README
index 379c390..3a38588 100644
--- a/kernel-shark/README
+++ b/kernel-shark/README
@@ -48,6 +48,7 @@ Example:
     cmake -D_DOXYGEN_DOC=1 -D_DEBUG=1 -D_INSTALL_PREFIX=/usr ../
 
 2.2.1 Use "make clean" if you want to delete all already compiled objects.
+This will also clean up all the files created by cmake.
 
 2.2.2 Use the script "cmake_clean.sh" if you want to delete all already
 compiled objects and all files generated by CMake.
-- 
2.20.1

